src/server/api/controllers/events.js

```diff
@@ -1,20 +1,18 @@
-import { couchDb } from '../../../shared/config';
+import { badgerBrain } from '../../../shared/config';
 import qs from 'qs';
 import moment from 'moment';
 import slugify from 'slug';
 import request from 'request';
 
 export default class EventsController {
   getEvents = (req, res) => {
-    fetch(couchDb.remote.protocol + (process.env.NODE_ENV === 'production' ? (couchDb.remote.host + ':' + couchDb.remote.port) : (couchDb.local.host + ':' + couchDb.local.port)) + '/events/_all_docs?include_docs=true')
+    fetch('http://127.0.0.1:3001/graphql?query={allEvents{id,slug,title,strapline,internalLinks{title,url},externalLinks{title,url},datetime{iso,date,month,monthSym,year},body{type,text},featureImageFilename}}')
```

Comment:
Yes there is - you can do a POST request with a query in body. Agree we should fix this.